sig-bug-triage-process.md

```diff
@@ -0,0 +1,55 @@
+## SIG Bug Triage Process
+
+SIGs should be triaging the issues with their matching label during every SIG
```

Comment:
Minor:
Prefer not to say "should". Better to say "...will triage issues with..." or else describe the expectations: "Time permitting, SIGs will triage..."